src/createinstancedialogpages.cpp

```diff
@@ -570,7 +571,8 @@ void GamePage::fillList()
       continue;
     }
 
-    if (!m_filter.matches(g->game->gameName())) {
+    if (!m_filter.matches(g->game->gameName()) &&
```

Comment:
I'm not sure this should be kept - The `gameName()` is now internal only, so using it to filter for users could lead to unexpected behavior.

Reply:
This area is just the filter search when creating a profile, I don't think it'd hurt anything to continue using the gameName alongside the displayName

For tale of two wastelands for example I can search "TTW" or "Tale of Two wastelands"